src/main/java/com/google/sps/servlets/UploadOnBlobstoreServlet.java

```diff
@@ -8,19 +8,18 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-
-@WebServlet("/blobstore-upload-url-gallery")
+@WebServlet("/upload")
 public class UploadOnBlobstoreServlet extends HttpServlet {
-  
+
   // Create an upload URL for the user to upload their images to BlobStore
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
 
-    // Redirects the user to /usercards servlet upon completion of 
+    // Redirects the user to /usercards servlet upon completion of
```

Comment:
completion of ...?

Reply:
"completion of image being processed in blobstore"
Done!